R/renderWordcloud2.R

```diff
@@ -4,15 +4,15 @@
 ##' Function for plotting wordcloud2 in shiny
 ##'
 ##' @usage
-##' wordcloud2Output(outputId, width = "100\%", height = "400px",
-##'                  clickedWordInputId = "selected_word")
+##' wordcloud2Output(outputId, width = "100\%", height = "400px")
+##' clickedWordInputId is automatically generated by: paste0(outputId, "_clicked_word"))
+##' 
```

Comment:
If `clickedWordInputId` is not a param in the function, it shouldn't in @usage part.

Please add these to @details part